backend/src/test/java/capstone/myfinancemanager/manager/service/TransactionServiceTest.java

```diff
@@ -64,6 +66,26 @@ class TransactionServiceTest {
             .pictureId("url")
             .build();
 
+    TransactionCreationDto transactionCreationDto1 = TransactionCreationDto.builder()
+            .description("Tanken")
+            .amount(27.0)
+            .transactionDate(Long.parseLong("1661866382913"))
+            .category("TestCategory")
+            .userEmail("test@test.com")
+            .isIncome(true)
+            .pictureId("url")
+            .build();
+
+    TransactionCreationDto transactionCreationDtoUpdate = TransactionCreationDto.builder()
```

Comment:
`TransactionCreationDto` and then transaction**Creation**Dto**Update** makes me wonder. Maybe we can find a better name? Is TransactionDto used in responses and TransactionCreationDto in requests? Then maybe TransactionResponseDto and TransactionRequestDto? Or maybe create an extra class for updates? 
Names are hard.. :)